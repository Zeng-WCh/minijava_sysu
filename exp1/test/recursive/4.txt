6+7-8+1+0+6+5-1+3-1-6-0-6+1+2+3+9+3+2+5-2+1-8+8+5-8-7-4+9+7