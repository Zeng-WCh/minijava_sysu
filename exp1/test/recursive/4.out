Input an infix expression and output its postfix notation:
03+9+9+0+1+6-8-3+5-8-4+3-2+2+4-0-4+2+8-5-5+5+6-7+6+8-0+2+6-
End of program.
